lib/typeprof/cli/cli.rb

```diff
@@ -175,5 +182,9 @@ def set_profiler
         StackProf.results
       end
     end
+
+    def generate_config_file
+      File.write('typeprof.conf.jsonc', File.read(File.join(__dir__, 'typeprof.conf.jsonc')))
```

Comment:
```suggestion
      File.write('typeprof.conf.jsonc', File.read(File.join(__dir__, 'typeprof.conf.jsonc')), mode: "wx")
```

I think it's a good idea to raise an error when trying to overwrite an existing file

Reply:
Thank you! This one is definitely much better.